packages/frontend/src/components/sign/SignTransferRetry.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import ArrowUpImage from '../../images/icon-arrow-up-green.svg';
+import RetryImage from '../../images/icon-retry-tx.svg';
+import FormButton from '../common/FormButton';
+
+const CustomContainer = styled.div`
+    max-width: 450px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #25282A;
+    text-align: center;
+
+    .title {
+        margin-top: 24px;
+
+        h2 {
+            font-weight: 900;
+            font-size: 22px;
+            color: #24272a;
+        }
+    }
+
+    && .text {
+        color: #72727A;
+        margin-top: 24px;
+    }
+
+    .buttons {
+        display: flex;
+        width: 100%;
+        margin-top: 40px;
+
+        button {
+            flex: 1;
+
+            &:last-of-type {
+                margin-left: 30px !important;
+
+                @media (min-width: 768px) {
+                    margin-left: 50px !important;
```

Comment:
Can we please just add `&&& {...}` at the top of the css so that we can remove `!important` - trying to actively remove all usage of `!important` in our css

Reply:
Done.